koku/masu/test/processor/test_tasks.py

```diff
@@ -406,6 +407,40 @@ def test_summarize_reports_processing_list_with_none(self, mock_update_summary):
 
         mock_update_summary.s.assert_called_once()
 
+    @patch("masu.processor.tasks.update_summary_tables")
+    def test_summarize_reports_processing_list_no_dates_month_middle(self, mock_update_summary):
+        """Test that the summarize_reports task is called once when a processing list
+        of reports with no start or end date provided."""
+        mock_update_summary.s = Mock()
+
+        reports_to_summarize = [{"manifest_id": id} for id in range(1, 5)]
+
+        current_date = datetime.now()
+        with patch(
+            "masu.external.date_accessor.DateAccessor.today",
+            return_value=datetime(current_date.year, current_date.month, 28),
```

Comment:
I use the current year and month of the test run here. I think these values should also be static, but wasn't sure if wanted to keep them dynamic.